libs/dh/wholesale/feature-settlement-reports/src/lib/tabs/dh-wholesale-settlements-reports-tabs-balance.component.ts

```diff
@@ -0,0 +1,100 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, Input, OnInit, inject } from '@angular/core';
+import { FormBuilder, ReactiveFormsModule } from '@angular/forms';
+import { WattButtonModule } from '@energinet-datahub/watt/button';
+import { WattCardModule } from '@energinet-datahub/watt/card';
+import { WATT_TABS } from '@energinet-datahub/watt/tabs';
+import { TranslocoModule } from '@ngneat/transloco';
+import { WattDatepickerModule } from '@energinet-datahub/watt/datepicker';
+import { WattFormFieldModule } from '@energinet-datahub/watt/form-field';
+import { graphql } from '@energinet-datahub/dh/shared/domain';
+import { Subject, takeUntil } from 'rxjs';
+import { Apollo } from 'apollo-angular';
+import { WattDropdownModule, WattDropdownOption } from '@energinet-datahub/watt/dropdown';
+import { ActorFilter } from '@energinet-datahub/dh/wholesale/domain';
+
+@Component({
+  standalone: true,
+  selector: 'dh-wholesale-settlements-reports-tabs-balance',
+  templateUrl: './dh-wholesale-settlements-reports-tabs-balance.component.html',
+  styleUrls: [
+    '../dh-wholesale-settlements-reports-tab.component.scss',
```

Comment:
suggestion: nice approach but with the twist that styling in this file *could* lead to side effects in this component. Can see that the common styling is related to `watt-card-title`. Let's copy the styling for now in this component's SCSS file. Later we can move it to a shared file. Or we can move it it a shared file already now. Could be in a `shared` sub-folder.